nixos/services/redis.nix

```diff
@@ -99,7 +88,7 @@ let
 
   telegrafInputs =
     let
-      mkPassword = name: getRedisPassword name (lib.const "\${REDIS_PASSWORD_${name}}");
+      mkPassword = name: getRedisPassword name (lib.const "\${REDIS_PASSWORD_${escape name}}");
```

Comment:
we'll need a refactoring here. I understand why you wanted to fold the if/else into the getRedisPassword function, however, this turns out to be too much inside/out wrapping things ...